_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,49 @@
+Something old, something new...
+
+
+Even though it’s only June, our HackIllinois Summer team is already well under way in planning the event for next February! We’ve been exploring tons of new possibilities for what the future of our yearly gathering will look like, and fine tuning the things that didn’t go as well last year.
```

Comment:
potential idea is publishing this article later in the summer when we have a more concrete idea on what new things will *probably* be implemented during the event - danger with this is that that could also still be too early, but I like the idea of hyping up new changes as this article does a little

Reply:
I totally agree that we shouldn't publish anything we aren't super sure about doing again, I'm working through a revision now that cleans these out a bit more